bin/minikube/launch.py

```diff
@@ -658,9 +671,9 @@ def initialize_database(
         release_name=postgres_release,
         namespace=namespace,
         db_host=postgres_host,
-        db_password=config.db_password,
+        db_admin_password=config.db_admin_password,
         db_name=config.db_name,
-        db_user=config.db_name,
+        db_admin_user=config.db_name,
```

Comment:
db_admin_user=config.db_name, -> ça seems very wrong to me.
I think that's kind of a typo

Reply:
i think there is a confusion between : 
- host
- instance name
- db name
- user name